sentysis-angular/src/app/form/form.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormsModule, ReactiveFormsModule } from '@angular/forms';
+
+import { YtUrl } from './form-url';
+import { FormService } from './form.service';
+
+@Component({
+  selector: 'app-form',
+  providers: [FormService],
+  templateUrl: './form.component.html',
+  styleUrls: ['./form.component.css']
+})
+export class FormComponent implements OnInit {
+
+  ytUrls!: YtUrl[];
+  youtubeForm;
+  videoId = '';
+
+
+  constructor(
+    private formBuilder: FormBuilder,
+    private formService: FormService
+  ) {
+    this.youtubeForm = this.formBuilder.group({
+      videoUrl: ''
+    });
+  }
+  ngOnInit() {
+  }
+
+  onSubmit(newYtUrl) {
+    if(typeof newYtUrl.videoUrl == 'string'){
+      let stringYtUrl = newYtUrl.videoUrl;
+      if(this.verifyUrl(stringYtUrl)) {
+        console.log('GANGSTA');
+        this.formService
+        .addYtUrl(newYtUrl)
+        .subscribe(ytUrl => this.ytUrls.push(ytUrl));
+
+      }
+      else {
+        console.log('WRONG');
+      }
+      this.youtubeForm.reset();
+    }
+    this.youtubeForm.reset();
+  }
+
+  // helper methods for onSubmit()
+  verifyUrl(ytUrl) {
```

Comment:
So what's the whole point of this method: `verifyUrl`?
Shouldn't this call be in the Flask server instead?

Reply:
Oh this is like confirming that it's an actual youtube url, so if youtube url then proceed with the POST but if not stop the process right there

Reply:
Good point. Good reasoning to me! 

I recommend using Regex for this: https://stackoverflow.com/questions/19377262/regex-for-youtube-url